packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -45,9 +45,9 @@ export default function ListViewSidebar() {
 	useShortcut( 'core/edit-post/toggle-list-view', () => {
 		// If the list view has focus, we know it is safe to close.
 		if (
-			listViewRef.current.contains(
-				listViewRef.current.ownerDocument.activeElement
-			)
+			listViewRef.current
+				.closest( '[role="region"]' )
```

Comment:
Was this change needed to make sure the "close" button is within this div? 

Can we instead just move the listViewRef to the div in line 62 of this file? The goal is to avoid relying on something that is not rendered by this component itself

Reply:
@youknowriad I gave it a try and updated the E2E test. How does this look?